src/extensions/example-extension/example-extension.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button, DynamicPageType, Icon, LensExtension } from "@lens/extensions"; // fixme: map to generated types from "extension-api.ts"
+import React from "react";
+import path from "path";
+
+let extension: ExampleExtension; // todo: provide instance from context
```

Comment:
Couldn't this be achieved by a static method on `ExampleExtension`?

Reply:
This solution was temporary